code/game/machinery/buttons.dm

```diff
@@ -47,4 +47,50 @@
 	var/id = 1
 
 	attack_paw(mob/user as mob)
-		return
\ No newline at end of file
+		return
+
+/obj/machinery/medical_help_button
+	name = "Medical attention required"
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "doorctrl0"
+	desc = "A button for alerting doctors that you require assistance."
+	var/id = null
```

Comment:
Why define it if you aren't using it in any way?

Reply:
copy paste from other working buttons.